src/rpcrawtransaction.cpp

```diff
@@ -334,12 +334,18 @@ UniValue createrawtransaction(const UniValue& params, bool fHelp)
             "      ,...\n"
             "    }\n"
 
+            "\n"
+            "    If you want to create OP_RETURN output instead of an address output, entries should be in the following format:\n"
```

Comment:
Prefer expounding on the proper use cases here to avoid people getting the impression it should be used incorrectly.